lib/location/method_gnss.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef METHOD_GNSS_H
+#define METHOD_GNSS_H
+
+int method_gnss_init(void);
+void method_gnss_event_handler(int event);
+void method_gnss_fix_work_fn(struct k_work *item);
+void method_gnss_timeout_work_fn(struct k_work *item);
+
+/** @brief Sets the GNSS configuration and starts GNSS.
+ *
+ * @details Currently supported configurations are fix interval, fix timeout and fix accuracy.
```

Comment:
Is this a necessary comment? I guess all parameters in the configuration should be supported?

Reply:
Removed the whole comment since it was not describing this function anymore.

Reply:
This hasn't been removed from the code